my-app/src/app/blog/carousel/carousel.component.ts

```diff
@@ -0,0 +1,53 @@
+import { Component, OnInit } from '@angular/core';
+import { DomSanitizer } from '@angular/platform-browser';
+
+@Component({
+  selector: 'carousel',
+  templateUrl: './carousel.component.html',
+  styleUrls: ['./carousel.component.css']
+})
+export class CarouselComponent implements OnInit {
+
+  constructor(private sanitizer: DomSanitizer) { }
+
+  ngOnInit() {
+    setInterval(() => {
+      this.onNext();
```

Comment:
I'm getting this error in the concole from the carousel changing

```
ErrorUtils caught an error: "Tried to listen to element of type click from Error: Cannot listen to an unde...". Subsequent errors won't be logged; see https://fburl.com/debugjs. 
```

Look into it when you have time, but this is low priority. You should never have errors in the console, unless they don't appear in production.